apps/desktop/e2e/automated/slash_screen.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { expect, test, ElectronApplication } from '@playwright/test';
+import { prepElectronApp } from '../__fixtures__/prep';
+
+let electronApp: ElectronApplication;
+
+test.beforeEach(async () => {
+  electronApp = await prepElectronApp();
+});
+
+test.afterAll(async () => {
+  await electronApp.close();
+});
+
+test.skip('record', async () => {
+  const splash = await electronApp.firstWindow();
+  await splash.pause();
+});
```

Comment:
I think we can safely remove all paused test cases.

Reply:
Pause was only there in record. Commented it out so that it won't affect in CI and can use whenever required.